libraries/standard/http/utest/README.md

```diff
@@ -0,0 +1,127 @@
+# Running Unit Tests
+
+1. In your GNU compatible environment, open a terminal.
+1. Install ruby: `sudo apt install ruby-full`
+1. Install lcov: `sudo apt install lcov`
+1. Go to the root directory of this repository.
+1. Update submodules (if you haven't already): `git submodule update --init --recursive` 
+1. Create build directory: `mkdir build && cd build`
+1. Run cmake while inside build directory: `cmake ..`
+1. Run this command to build and run the tests: `make coverage`
+1. Go to `build` and open `http_utest_out.txt` to view logs
```

Comment:
I think having the readme in every folder is a bit of an overkill
if something ever changes (typo, bug, new step, removing a step etc..) we will have to go to all folders it could grow up to a lot , and make the changes in each one.
I would suggest in a folder called Doc , UnitTestDoc etc

Reply:
Yah it has been quite a hassle editing both README's lol